main.js

```diff
@@ -50,6 +52,44 @@ function createWindow(commandLineArguments, workingDirectory) {
 
     // Create the browser window.
     let mainWindow = new BrowserWindow({ width: 800, height: 600, icon: path.join(__dirname, "images", "Oni_128.png") })
+    let menu = defaultMenu(app, shell);
+
+  menu.unshift ({
+    label: 'Oni',
```

Comment:
I tested this on Windows and it looks a bit unnatural - I think we might want `File` on Windows, and `Oni` on OSX/Linux